cloud4all.html

```diff
@@ -0,0 +1,151 @@
+        <ul>
+          <li>Statistical Matchmaker:
+            <ul>
+              <li>
+                <a href="https://github.com/REMEXLabs/GPII-Statistical-Matchmaker-Analysis/">analysis component</a>:
+                machine-learning algorithms
+              </li>
+              <li>
+                <a href="https://github.com/REMEXLabs/GPII-Statistical-Matchmaker">runtime component</a>:
+                the component that gets deployed in order to process matchmaking requests
+              </li>
+              <li>
+                <a href="https://github.com/REMEXLabs/GPII-Statistical-Matchmaker-Data">training data</a>:
+               used by the analysis component to create input for the runtime component
+              </li>
+            </ul>
+          </li>
+          <li>Rule-Based Matchmaker:
+            <ul>
+              <li>
+                <a href="https://github.com/NickKaklanis/RuleBased_MatchMaker">Rule-Based Matchmaker</a>
+              </li>
+              <li>
+                <a href="https://github.com/NickKaklanis/RuleBasedMatchMaker_RESTful_WS_Maven"><abbr>REST</abbr>ful web-service</a>
+              </li>
+              <li>
+                <a href="https://github.com/AggelikiKonstadinidou/rule-editor-portlet">Rule Editor for the Rule-Based Matchmaker</a>.
+                <!-- This repo does not have a licence. -->
+              </li>
+            </ul>
+          </li>
+        </ul>
+
+        <p>Other components:</p>
+        <ul>
+          <li>
+            <a href="https://github.com/NickKaklanis/ServiceSynthesizerTool_RESTful_WS_Maven">Service Synthesizer Tool (<abbr>SST</abbr>)</a>:
+            tool that combines existing Web Services into new ones
+          </li>
+          <li>
+            <a href="https://github.com/juanmanuel-fdez/Cloud4All---Context-Aware-Server">Context-Aware Server</a>:
+            <!-- previous address: 
+              <a href="https://github.com/barcelonadigital/Cloud4All---Context-Aware-Server">Context-Aware Server</a>:
+            -->
+            platform for collecting and processing sensor data and sending them
+            to the <abbr>GPII</abbr> Flow Manager
+          </li>
+          <li>
+            <a href="https://github.com/barcelonadigital/Cloud4All---Context-Aware-Server">Context-Aware server</a>:
+            a component that collects and processes data from sensors and sends 
+            the processed data to the Flow Manager.
+          </li>
+          <li>
+            <a href="https://github.com/JChaconTechnosite/Cloud4ALL-MiniMatchMaker">MiniMatchMaker</a>
+            (also known as &ldquo;Spinal Matchmaker&rdquo;) for Android.
+          </li>
+          <li>
+            <a href="https://github.com/JChaconTechnosite/Cloud4ALL-Enviromental-reporter">Environmental Reporter for Android</a>.
+          </li>
+          <li>
+            <a href="https://github.com/adelga/AndroidCloud4All">Cloud4all Android implementation</a>.
```

Comment:
Yes, it seems like this link should be changed to the [official Android repository](https://github.com/gpii/android), since FVE's features have been merged into it, right @javihernandez?